doc/onboarding.md

```diff
@@ -100,18 +107,20 @@ onboarding session.
         (especially if it just has nits left).
   * Approving a change
     * Collaborators indicate that they have reviewed and approve of the
-        the changes in a pull request by commenting with `LGTM`, which stands
-        for "looks good to me".
-    * You have the power to `LGTM` another collaborator's (including TSC/CTC
-        members) work.
-    * You may not `LGTM` your own pull requests.
-    * You have the power to `LGTM` anyone else's pull requests.
+        the changes in a pull request using Github’s approval interface
+    * Some people like to comment `LGTM` (“Looks Good To Me”)
+    * You have the power to approve any other collaborator's work.
+    * You may not approve your own pull requests.
+    * When explicitly using `Changes requested`, show empathy – comments will
+      usually be addressed even if you don’t use it.
+      * If you do, it is nice if you are available later to check whether your
+        comments have been addressed
 
   * What belongs in node:
-    * opinions vary, but I find the following helpful:
-    * if node itself needs it (due to historic reasons), then it belongs in node
-      * that is to say, url is there because of http, freelist is there because of http, et al
-    * also, things that cannot be done outside of core, or only with significant pain (example: async-wrap)
+    * Opinions vary – it’s good to have a broad collaborator base for that reason!
+    * If node itself needs it (due to historic reasons), then it belongs in node
+      * That is to say, url is there because of http, freelist is there because of http, et al
```

Comment:
period after `et al.` (it's in the phrase not an end of sentence period)
or even replace with the more common `etc.` (also `.` is part of the word)